src/style.css

```diff
@@ -0,0 +1,19 @@
+
+.red {
+  background-color: red;
+  height: 100px;
```

Comment:
do not repeat styles
will be better to create one general class for block and add height style to it